templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Contributing
+
+[fork]: https://github.com/ORG/REPO/fork
+[pr]: https://github.com/ORG/REPO/compare
+[style]: STYLEGUIDE
+[code-of-conduct]: CODE_OF_CONDUCT.md
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
+
+## Submitting a pull request
+
+0. [Fork][fork] and clone the repository
+0. Create a new branch: `git checkout -b my-branch-name`
+0. Make your change and remember to add tests
+0. Build the project locally and run local tests
+0. Push to your fork and [submit a pull request][pr]
+0. Pat your self on the back and wait for your pull request to be reviewed and merged.
+
+Here are a few things you can do that will increase the likelihood of your pull request being accepted:
+
+- Follow the [style guide][link to styleguide].
+- Write tests.
+- Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, submit them as separate pull requests.
+- Write [good commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+
+## Resources
+
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [Using Pull Requests](https://help.github.com/articles/about-pull-requests/)
+- [GitHub Help](https://help.github.com)
```

Comment:
facebook/react link is not working
freeCodecamp link is re-directed
the rests are detailed